exp/util/secret/consts.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Similar with these `util/secret` helpers, if they are the same as the cloud-init based bootstrap provider, might be worth it to just import those instead.

Reply:
Removed `util/secret` from `exp`, and just used one in the upper-level.